### PR TITLE
[Merged by Bors] - feat: `f⁻¹` is continuous iff `f` is

### PR DESCRIPTION
**Pull request by YaelDillies**
... and similar results for lipschitzness.

From LeanCamCombi

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — COMMENTED** [review]
Seems reasonable to me - but somebody who knows this part of the library better should take a look as well.
maintainer merge

File: Mathlib/Topology/Algebra/Group/Basic.lean [hunk 1]
```diff
@@ -395,6 +395,23 @@ theorem inv_closure : ∀ s : Set G, (closure s)⁻¹ = closure s⁻¹ :=
...
+lemma continuousAt_inv_iff : ContinuousAt f⁻¹ x ↔ ContinuousAt f x :=
+  (Homeomorph.inv G).comp_continuousAt_iff _ _
+
+@[to_additive (attr := simp)]
+lemma continuousOn_inv_iff : ContinuousOn f⁻¹ s ↔ ContinuousOn f s :=
+  (Homeomorph.inv G).comp_continuousOn_iff _ _
+
+alias ⟨Continuous.of_inv, _⟩ := continuous_inv_iff
+alias ⟨ContinuousAt.of_inv, _⟩ := continuousAt_inv_iff
+alias ⟨ContinuousOn.of_inv, _⟩ := continuousOn_inv_iff
```
Comment (sgouezel) [inline Mathlib/Topology/Algebra/Group/Basic.lean:413]: to_additive attributes?

**Review by sgouezel — COMMENTED** [review]
bors d+

**Comment by YaelDillies** [comment]
Sorry, was too fast!

bors merge

**Pull request closed without merging**